src/main/java/org/sigmah/server/servlet/exporter/template/IndicatorEntryExcelTemplate.java

```diff
@@ -126,6 +126,8 @@ private void createDetailSheet(final IndicatorDTO indicator) {
 		int rowIndex = -1;
 
 		final List<PivotTableData.Axis> leaves = data.getEntryMap().get(indicator.getId()).getRootColumn().getLeaves();
+        final List<PivotTableData.Axis> leavesyear=data.getEntryMap().get(indicator.getId()+1).getRootColumn().getLeaves();
```

Comment:
@manishvishnoi2 : This needs an indentation fix before the pull request can be merged.